sunpy/data/_sample.py

```diff
@@ -116,9 +116,9 @@ def get_sample_file(filename, url_list, show_progress=True, overwrite=False,
     else:
         uncompressed_filename = filename
     # check if the (uncompressed) file exists
-    if not overwrite and os.path.isfile(os.path.join(sampledata_dir,
-                                                     uncompressed_filename)):
-        return os.path.join(sampledata_dir, uncompressed_filename)
+    if not overwrite and Path(str(Path.home().joinpath(sampledata_dir,
```

Comment:
Also why the `home()` here? `(Path(sampledata_dir) / uncompressed_filename).is_file()` mayeb?

Reply:
Yeah I was wondering if we should use the fancier method to join paths.

Reply:
yeah i have made the correction